src/main/java/org/alfresco/repo/search/impl/solr/SolrBackupClient.java

```diff
@@ -63,7 +63,7 @@
 
     private SOLRAdminClient solrAdminClient;
     

```

Comment:
Only reason i didnt update it as it was causing me grief on spring injection.
I can amend this but then we need to validate that it is working against solr4, solr6 etc.
If we can avoid it for now, it would be good.